src/views/account-detail/index.js

```diff
@@ -194,39 +194,44 @@ class AccountLatest extends Component {
-                                '<Td class="' + TdClass + '"> <div class="qty">' + value / 1000000 + ' <span>POKT</span></div> <div class="status">' + tx.type.toLowerCase() + '</div> </Td>\n' +
-                                '<Td class="' + TdClass + ' block-align">' + tx.height + '</Td>\n' +
-                                '<Td class="' + TdClass + '"> <a id="txHashElement' + idCounter + '"> ' + txHash + ' </a> </Td>\n' +
-                                '</Tr>';
-
-                            section.insertAdjacentHTML('beforeend', txTemplate);
-                            // Add onClick event to the clickable element
-                            const toTxDetail = document.getElementById(`txHashElement${idCounter}`);
-
-                            if (toTxDetail) {
-                                toTxDetail.addEventListener("click", () => { this.pushToTxDetail(txHash) });
-                            }
-                            idCounter++;
-                        } else {
-                            console.dir(attributes, { depth: null });
+                    events = JSON.parse(tx.tx_result.log)[0].events;
+                } else if (tx.tx_result.events) {
+                    events = tx.tx_result.events
+                }
+
+                if (events[1].type === "transfer") {
+                    const attributes = events[1].attributes;
+                    if (attributes[1].key === "amount") {
+                        const value = attributes[1].value.replace("upokt", "");
+
+                        const txHash = tx.hash;
+                        const imageSrc = tx.type.toLowerCase() === "sent" ? sentImgSrc : receivedImgSrc;
+                        const TrClass = document.getElementById("tr-element").className;
+                        const TdClass = document.getElementById("td-element").className;
+
+                        const txTemplate = '<Tr class="' + TrClass + '">\n' +
+                            '<Td class="' + TdClass + '"> <img src=' + imageSrc + ' alt="' + tx.type.toLowerCase() + '" /> </Td>\n' +
+                            '<Td class="' + TdClass + '"> <div class="qty">' + value / 1000000 + ' <span>POKT</span></div> <div class="status">' + tx.type.toLowerCase() + '</div> </Td>\n' +
+                            '<Td class="' + TdClass + ' block-align">' + tx.height + '</Td>\n' +
+                            '<Td class="' + TdClass + '"> <a id="txHashElement' + idCounter + '"> ' + txHash + ' </a> </Td>\n' +
+                            '</Tr>';
+
+                        section.insertAdjacentHTML('beforeend', txTemplate);
+                        // Add onClick event to the clickable element
+                        const toTxDetail = document.getElementById(`txHashElement${idCounter}`);
+
+                        if (toTxDetail) {
+                            toTxDetail.addEventListener("click", () => { this.pushToTxDetail(txHash) });
```

Comment:
Oof, I had not seen this 😅 and it's a bit bad... (The reason being this is using direct manipulation of HTML instead of just using React) But considering we might rebuild the app soon we can let this pass for now. Not too happy about this though haha

Reply:
It was my first time touching react when I did this code block

Reply:
Gotcha, that's okay 😄